### ISSUE TITLE
Graph Plots

### ISSUE THREAD

**Issue by omidb** [quantifind/wisp#27]
Thanks for awesome lib,
Does Wisp comes with graph visualization? like those in d3 ?

**Comment by AustinBGibbons** [comment]
Hey Omid, unfortunately today we only support canonical 2D charts, and no graph/network visualization (but would love to in the future - perhaps through evolving vega support)

**Comment by AustinBGibbons** [comment]
Hey Omid, unfortunately today we only support canonical 2D charts, and no graph/network visualization (but would love to in the future - perhaps through evolving vega support)

**Comment by omidb** [comment]
Yes, that would be a good idea, I know that "sparkle project" also using vega. I tried to look at your code, it's very dependent on Highchart, that would be nice if you can provide me with some structure information about the project, then I may find some time to merge vega with wisp.

**Comment by AustinBGibbons** [comment]
Yes - both the Highcharts implementation and repl API are highcharts focused.

We originally tried to sketch the type system around using other interfaces. You can see the work I started on vega here: https://github.com/quantifind/wisp/tree/austin/vega

Right now by implementing `plot` you can use any library - but you don't get support for things like line, column, bar, etc. - as those are highcharts specific methods. (for example, a graph implementation may not support them at all)

We can follow up by email if you'd like to discuss further - austin [at] name-of-the-company.com
